package.json

```diff
@@ -5,7 +5,24 @@
   "author": "CZI <edu-frontend-infra@chanzuckerberg.com>",
   "homepage": "https://github.com/chanzuckerberg/edu-design-system",
   "license": "MIT",
-  "main": "lib/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./lib/index.cjs"
+    },
+    "./index.css": {
+      "import": "./lib/index.css",
+      "require": "./lib/index.css"
+    },
+    "./fonts.css": {
+      "import": "./lib/tokens/fonts.css",
+      "require": "./lib/tokens/fonts.css"
+    },
+    "./tailwind.config.mjs": {
+      "import": "./tailwind.config.mjs"
+    }
+  },
```

Comment:
@ahuth Not sure if I'm using conditional exports right but seems that we have to explicitly state which files are allowed to be exported

Reply:
Yeah, that's true. I kind of forgot that there was more than just the index.js files (such as fonts, css, tailwind, etc).

Does this seem like it's too cumbersome, or is it okay?

Reply:
It's fine unless consumers want to use some of the internal components or the utils, which we haven't come across yet